config/config.go

```diff
@@ -156,8 +156,9 @@ type GDPR struct {
 	Timeouts                GDPRTimeouts `mapstructure:"timeouts_ms"`
 	NonStandardPublishers   []string     `mapstructure:"non_standard_publishers,flow"`
 	NonStandardPublisherMap map[string]int
-	TCF2                    TCF2 `mapstructure:"tcf2"`
-	AMPException            bool `mapstructure:"amp_exception"`
+	TCF2                    TCF2     `mapstructure:"tcf2"`
+	AMPException            bool     `mapstructure:"amp_exception"`
+	EEACountries            []string `mapstructure:"eea_countries"`
```

Comment:
Please add a comment to clarify that EEA = European Economic Area and explain we will always enforce GDPR for requests with a geo which match this list.

Reply:
done